2-WebApp-graph-user/2-5-HybridFlow/README.md

```diff
@@ -0,0 +1,480 @@
+  
+    1. On Windows, run PowerShell as **Administrator** and navigate to the root of the cloned directory
+    1. In PowerShell run:
+
+       ```PowerShell
+       Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope Process -Force
+       ```
+
+    1. Run the script to create your Azure AD application and configure the code of the sample application accordingly.
+    1. For interactive process - in PowerShell run:
+
+       ```PowerShell
+       cd .\AppCreationScripts\
+       .\Configure.ps1 -TenantId "[Optional] - your tenant id" -Environment "[Optional] - Azure environment, defaults to 'Global'"
+       ```
+
+       > Other ways of running the scripts are described in [App Creation Scripts guide](./AppCreationScripts/AppCreationScripts.md)
+       > The scripts also provide a guide to automated application registration, configuration and removal which can help in your CI/CD scenarios.
+
+  </details>
+
+### Manual Steps
+
+ > Note: skip this part if you've just used Automation steps
+
+Follow the steps below for manually register and configure your apps
+
+<details>
+   <summary>Expand this section if you want to use this automation:</summary>
+  1. Sign in to the [Azure portal](https://portal.azure.com).
+  1. If your account is present in more than one Azure AD tenant, select your profile at the top right corner in the menu on top of the page, and then **switch directory** to change your portal session to the desired Azure AD tenant.
+
+#### Register the service app (TodoListService-aspnetcore-webapi)
+
+  1. Navigate to the [Azure portal](https://portal.azure.com) and select the **Azure Active Directory** service.
+  1. Select the **App Registrations** blade on the left, then select **New registration**.
+  1. In the **Register an application page** that appears, enter your application's registration information:
+     * In the **Name** section, enter a meaningful application name that will be displayed to users of the app, for example `HybridFlow-aspnetcore`.
+  1. Under **Supported account types**, select **Accounts in this organizational directory only**
+  1. Click **Register** to create the application.
+  1. In the app's registration screen, find and note the **Application (client) ID**. You'll need to use this value in your app's configuration files.
+  1. In the app's registration screen, select **Authentication** in the menu.
+     * If you don't have a platform added, select **Add a platform** and select the **Web** option.
+     * In the **Redirect URIs** section, enter the following redirect URIs.
+        * `https://localhost:44321/signin/`
+     * Click on the **Add a platform** button in the **Platform configurations** section of the page
+       * Select the **Single-page application** button and enter `https://localhost:7089/` as the **Redirect URI** and click the **Configure** button
+     * Under the **Implicit grant and hybrid flows** section enable both the **Access tokens** and **ID tokens** options.
```

Comment:
>**Access tokens** [](http://example.com/codeflow?start=77&length=17)

is this really required? its a big security snafu

Reply:
Should not be required for MSAL.js v2 and hybrid spa flow.

Reply:
Thanks for heads up.

Also tested. No issues.

Removed!